src/test/java/org/jenkinsci/plugins/workflow/graphanalysis/NoOpChunkFinder.java

```diff
@@ -10,6 +10,7 @@
  * All {@link FlowNode}s will this result in calling {@link SimpleChunkVisitor#atomNode(FlowNode, FlowNode, FlowNode, ForkScanner)}
  */
 public class NoOpChunkFinder implements ChunkFinder {
+
```

Comment:
In general I would prefer to avoid gratuitous formatting changes as part of a PR which is not _just_ formatting changes (which should rather be one to enable Spotless).